code/game/gamemodes/cult/runes.dm

```diff
@@ -400,7 +400,7 @@ var/list/sacrificed = list()
 	if(user.loc==src.loc)
 		var/mob/living/carbon/human/L = user
 		user.say("Fwe[pick("'","`")]sh mah erl nyag r'ya!")
-		user.visible_message("<span class='warning'>[user]'s eyes glow blue as \he freezes in place, absolutely motionless.</span>", \
+		user.visible_message("<span class='warning'>[user]'s eyes glow blue as \he collapses limply to the floor, their breathing slowing.</span>", \

```

Comment:
the phrasing here seems off. not quite sure why. or what was wrong with the old one for that matter.

Reply:
gameplay-narrative separation.  When you ghost, your body collapses to the floor, and I wanted to reflect that better than 'remaining motionless.

Reply:
`falls limply to the [isspace(get_turf(user)) ? "floor" :  "[get_turf(user)]"], \his breath slowing.`

Reply:
I don't think that does what you think it does.